care/utils/queryset/communications.py

```diff
@@ -0,0 +1,22 @@
+from care.hcx.models.communication import Communication
+from care.users.models import User
+from care.utils.cache.cache_allowed_facilities import get_accessible_facilities
+
+
+def get_communications(user):
+    queryset = Communication.objects.all()
+    if user.is_superuser:
+        pass
+    elif user.user_type >= User.TYPE_VALUE_MAP["StateLabAdmin"]:
+        queryset = queryset.filter(claim__policy__patient__facility__state=user.state)
+    elif user.user_type >= User.TYPE_VALUE_MAP["DistrictLabAdmin"]:
+        queryset = queryset.filter(
+            claim__policy__patient__facility__district=user.district
```

Comment:
@vigneshhari As I'm doing these many joins, does this have any performance implications?

Reply:
Should be fine, since we cant really do anything else here. until we build a better permission system, let this stay .